packages/core/fixtures/map-context.fixtures.ts

```diff
@@ -64,6 +65,11 @@ export const MAP_CTX_LAYER_MAPBLIBRE_STYLE_FIXTURE: MapContextLayerMapLibreStyle
     styleUrl: "http://my.host.com/maplibre/style.json",
     accessToken: "abcdefgh",
   });
+export const MAP_CTX_LAYER_MVT_FIXTURE: MapContextLayerMvt = deepFreeze({
+  type: "mvt",
+  url: "https://data.geopf.fr/tms/1.0.0/PLAN.IGN",
+  accessToken: "abcdefgh",
```

Comment:
maybe we don't really need a new type here, and instead use the `xyz` type, and add a "tileFormat" parameter to it

So for this layer it would be:
```js
{
  type: "xyz",
  url: "https://data.geopf.fr/tms/1.0.0/PLAN.IGN/{z}/{x}/{y}.pbf",
  tileFormat: "mvt",
  accessToken: "abcdefgh",
}
```